dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1499,7 +1499,11 @@ restoreServer()
     return $rc
   fi
   if [ $rc -ne 0 ]; then
-    issueMessage --message:info.checkpoint.restore.failed "${X_LOG_DIR}/checkpoint/restore.log"
+    if [ -n "${CRIU_RESTORE_DISABLE_RECOVERY}" ]; then
```

Comment:
If we are elevating this to an external then I think we should require the value to be set to "true" to disable.